types/index.d.ts

```diff
@@ -784,6 +784,12 @@ declare namespace RamdaAdjunct {
         */
         allSettledP<T>(iterable: Iterable<T>): Promise<Array<SettledPromise<T>>>;
 
+        /**
+         * anyP returns a promise that is fulfilled by the first given promise to be fulfilled,
```

Comment:
This is still different then the description in `anyP.js`. We have a convention to keep the description exactly the same.

```js
/**
 * Returns a promise that is fulfilled by the first given promise to be fulfilled,
 * or rejected with an array of rejection reasons if all of the given promises are rejected.
 *
....